pyclesperanto_prototype/_tier8/_AffineTransform3D.py

```diff
@@ -250,6 +250,21 @@ def shear_in_x_plane(self,angle_y_in_degrees: float = 0, angle_z_in_degrees: flo
         ]))
         return self
 
+    def scale_in_z_with_pixel_size(self, angle_in_degrees: float = 0, pixel_size_xy: float = 0, pixel_size_z: float = 0 ):
```

Comment:
We typically don't have variables called `pixel_size_xy` in clesperanto. We use `pixel_size_x` and `pixel_size_y` separately to be flexible.